plutus-ledger/src/Ledger/Scripts.hs

```diff
@@ -55,7 +55,7 @@ module Ledger.Scripts(
 
 import qualified Prelude                              as Haskell
 
-import           Codec.Serialise                      (serialise, Serialise)
+import           Codec.Serialise                      (Serialise, serialise)
 import           Control.DeepSeq                      (NFData)
 import           Control.Monad.Except                 (MonadError, runExcept, throwError)
 import           Crypto.Hash                          (Digest, SHA256, hash)
```

Comment:
Again, this has nothing to do with costing.  I presume it's happened because some formatting tool has been run over the entire codebase.  Do we have a policy about altering unrelated code in PRs?  I've done that myself a couple of times and it always made me feel uncomfortable.

Reply:
I run stylish, I commit.

Reply:
If you're running the same version of stylish as everyone else, then it shouldn't affect unrelated code, since that should already have been formatted :thinking: where do you get your `stylish-haskell` from?

Reply:
I copy/paste the nix expression from a CONTRIBUTING IIRC